docs/src/getting-started/quickstart.md

```diff
@@ -100,46 +98,39 @@ To quickly setup and bootstrap the PostgreSQL database that we'll need, we'll us
 
 We can quickly create a bootstrapped database and start the Fuel indexer service by running the following command:
 
-> IMPORTANT: Ensure that any local PostgreSQL instance that is running on port `5432` is stopped.
+> IMPORTANT: Below we're specifying our Postgres hostname as `--postgres-host postgresql`, but you will need to be specific to your own Postgres instance details (see `forc index start --help` for more details). You can try using the `--embedded-database` flag in order to quickly use an embedded instance of Postgres, but this is flaky and often depends on what platform you're using.
 
 ```bash
-forc index start \
-    --embedded-database
```

Comment:
`--embedded-database` is simply just not reliable enough to include this in the Quickstart.